### ISSUE TITLE
Loss function for CV?

### ISSUE THREAD

**Issue by alejandroschuler** [rieszreg/rieszreg#2]
when doing cross-validation (with sklearn utilities), how does CV operate? For example, if I initialize two learners like:

```
booster1 = RieszBooster(
    estimand=TSM(level=1),
    loss=KLLoss(max_eta=50),
    n_estimators=300,
    learning_rate=0.05,
    max_depth=3,
).fit(df)
```

```
booster2 = RieszBooster(
    estimand=TSM(level=1), # uses default loss (sq error I beleive)
    n_estimators=300,
    learning_rate=0.05,
    max_depth=3,
).fit(df)
```

and I want to do model selection between them with sklearn utilities, how would this operate? Ideally, the sklearn utility would 1) check whether an optional loss/score arg was passed and use that if so 2) if not, check whether all learners use the same loss and default to that if so and 3) raise error otherwise, asking user to provide the optional arg. Can we get clarity on this or otherwise implement this behavior?
